File: import/src/main/java/de/helfenkannjeder/helfomat/picture/DownloadFailedException.java
package de.helfenkannjeder.helfomat.picture;

/**
 * @author Valentin Zickner
 */
public class DownloadFailedException extends Exception {

    DownloadFailedException() {
    }

    DownloadFailedException(Exception e) {
        super(e);
    }
}

File: helfomat-core/src/main/kotlin/de/helfenkannjeder/helfomat/core/picture/DownloadFailedException.kt
package de.helfenkannjeder.helfomat.core.picture;

/**
 * @author Valentin Zickner
 */
public class DownloadFailedException extends Exception {

    public DownloadFailedException() {
    }

    public DownloadFailedException(Exception e) {
        super(e);
    }
}
